scripts/md5list.py

```diff
@@ -33,9 +33,9 @@
 			md5call = ["md5", "-r"]
 		else:
 			md5call = ["md5sum"]
-		md5call = md5call + ["src/"+string.strip(separate[1])]
+		md5call = md5call + ["src/"+separate[1].strip()]
 		md5sum = subprocess.Popen(md5call, stdout=subprocess.PIPE).communicate()[0]
-		md5sum = string.split(md5sum)
+		md5sum = md5sum.split()
 		res = "%-32s = %s\n" % (separate[1], md5sum[0])
```

Comment:
I am very sorry for this mistake. At this point `b'8d10ea15d5a180873ce4195011fe9f5f'` will be written, and OpenTTD will fail to read it:
```
dbg: [grf] Malformed MD5 checksum specified for: tttheme2.mid (in /usr/share/games/openttd/gm/openmsx-0.3.1/openmsx.obm)
```

`md5sum[0].decode()` is needed.